packages/react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -300,8 +300,10 @@ export class GroupedListBase extends React.Component<IGroupedListProps, IGrouped
 
   private _getGroupHeight = (group: IGroup): number => {
     const rowHeight = this.props.compact ? COMPACT_ROW_HEIGHT : ROW_HEIGHT;
+    const rowCount =
+      group.children && group.children.length > 0 ? group.children.length : this._getGroupItemLimit(group);
```

Comment:
Thanks for looking into this and finding a fix! One refinement: I think if both `group.children` and `group.getGroupItemLimit` are provided, it still ought to call `group.getGroupItemLimit` instead of using the number of children.

Reply:
Also it might be clearer to move all the new logic into `_getGroupItemLimit`

Reply:
@ecraig12345 Thanks for a feedback. That sounds better. I'll move it to `_getGroupItemLimit`